source/agora/common/BlockStorage.d

```diff
@@ -567,7 +565,7 @@ public class BlockStorage
             {
                 idx_file = File(file_name, "rb");
 
-                size_t record_size = (size_t.sizeof * 2 + HASH_SIZE);
+                size_t record_size = (size_t.sizeof * 2 + Hash.sizeof);
```

Comment:
Could we add a comment why the `size_t.sizeof * 2 + Hash.sizeof`? It's very magic..